src/Integration/Persistence/ISolutionBindingDataWriter.cs

```diff
@@ -29,7 +29,7 @@ internal interface ISolutionBindingDataWriter
         /// Writes the binding information
         /// </summary>
         /// <returns>Has file been saved</returns>
-        bool Write(string configFilePath, BoundSonarQubeProject binding, Action<string> onSuccessfulFileWrite);
```

Comment:
The action parameter was what used to stored in the `WriteSettings.OnSuccessfulFileWrite` field.
It wasn't being used so I removed it.